qcelemental/tests/test_importing.py

```diff
@@ -39,6 +39,50 @@ def test_which_import_f_raisemsg():
     assert str(e.value).endswith("Python module 'evilpint' not found in envvar PYTHONPATH. Install `evilpint`.")
 
 
+def test_which_import_t_submodule():
+    ans = qcel.util.which_import('pint.util')
+    assert ans.split(os.path.sep)[-1] == 'util.py'
+
+
+def test_which_import_t_submodule_altsyntax():
+    ans = qcel.util.which_import('.util', package='pint')
+    assert ans.split(os.path.sep)[-1] == 'util.py'
+
+
+def test_which_import_t_bool_submodule():
+    ans = qcel.util.which_import('pint.util', return_bool=True)
+    assert ans is True
+
+
+def test_which_import_f_submodule():
+    ans = qcel.util.which_import('evilpint.util')
+    assert ans is None
+
+
+def test_which_import_f_submodule_altsyntax():
+    ans = qcel.util.which_import('.util', package='evilpint')
```

Comment:
to test something new, perhaps you want `which_import('.evilutil', package='pint')`

Reply:
We could add this, yes. It would be testing a very different case, however, that won't yield a `ModuleNotFoundError`, since `pint` would exist, even though the `.evilutil` submodule doesn't.